modules/post-installation-adding-nutanix-failure-domains-compute-machines.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/adding-nutanix-failure-domains.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="post-installation-adding-nutanix-failure-domains-compute-machines_{context}"]
+= Distributing compute machines across failure domains
+
+You can distribute compute machines across Nutanix failure domains by:
+
+* Modifying existing compute machine sets.
+* Creating new compute machine sets.
+
+The following procedure details how to distribute compute machines across failure domains by modify existing control plane machines sets. For more information on creating a control plane machine set, see xref:../machine_management/creating_machinesets/creating-machineset-nutanix.adoc#creating-machineset-nutanix[Creating a compute machine set on Nutanix].
+
+.Prerequisites
+
+* You have added the failure domains to the cluster's Infrastructure custom resource definition (CRD).
```

Comment:
Should change to:

You have added the failure domains configuration to the cluster's Infrastructure custom resource (CR).

Reply:
Done. Thank you.